.github/PULL_REQUEST_TEMPLATE/deployment-template.md

```diff
@@ -0,0 +1,17 @@
+# Deployment: <!-- Deployment title: contract or feature (e.g. `ManagedPool`) -->
+
+## Deployment tasks checklist:
+
+- [ ] Contract(s) deployed to all target networks / testnets
+- [ ] TX IDs are properly generated
+- [ ] Action IDs are generated after deployment (if applicable) <!-- This includes contracts deployed by contracts (e.g. mock pools deployed by factories) -->
+- [ ] Contracts are verified in every network <!-- This includes contracts deployed by contracts (e.g. mock pools deployed by factories) -->
+
+## Code checklist:
+
+- [ ] The diff is legible and has no extraneous changes
+- [ ] The base branch is either `master`, or there's a description of how to merge
```

Comment:
Would this ever not be master? Maybe we don't need a code checklist section for this one, as there shouldn't be any code changes at this point.

Reply:
I agree; I thought about keeping the template just to enforce those two things explicitly, and force the submitter to double check.

If you think it's _too_ redundant, maybe we can just remove the checklist.

Reply:
Done in https://github.com/balancer-labs/balancer-v2-monorepo/pull/2168/commits/3121c9b4b7f773aba3a37c387802bbbac845bb0a.